File: testNombre.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

import mypyext

a=mypyext.Nombre()
a.setV(2.5)
print(str(a.getV()))
print(str(a.carre()))

a.setNom("Truc")
print(a.getNom())

b=mypyext.Nombre(3.14,"pi")

File: testNombre.rb
#!/usr/bin/env ruby


require './myrbext'

a=Myrbext::Nombre.new
a.setV(2.5)
puts a.getV
puts a.carre

b=Myrbext::Nombre.new(3.14,"pi")
puts "#{b.getNom} #{b.getV}"
